Makefile

```diff
@@ -223,6 +227,7 @@ kustomize: $(KUSTOMIZE) ## Build a local copy of kustomize.
 setup-envtest: $(SETUP_ENVTEST) ## Build a local copy of setup-envtest.
 controller-gen: $(CONTROLLER_GEN) ## Build a local copy of controller-gen.
 conversion-gen: $(CONVERSION_GEN) ## Build a local copy of conversion-gen.
+conversion-verifier: $(CONVERSION_VERIFIER) ## Build a local copy of conversion-verifier.
```

Comment:
where is it being used? Should we add it to verify?

Reply:
Done!